pythnet-price-feeds/evm.md

```diff
@@ -18,6 +18,13 @@ Please see the github readme for additional information on this service.
 
 Pyth is currently available on the following EVM-based chains:
 
+## Example - Oracle Swap 
+
+[Oracle Swap](https://github.com/pyth-network/pyth-crosschain/tree/main/target_chains/ethereum/examples/oracle_swap) is an end to end example that implements an AMM that uses Pyth prices to operate. It contains both the contract, and
```

Comment:
couple nitpicks here:

you always hyphenate "end-to-end"

I would also suggest splitting into multiple sentences for better readability "... end-to-end example application. This application is an AMM that allows users to swap two assets at the Pyth-provided exchange rate."

Also in general, i suggest rewriting sentences that start with "it" to use something like "the X" or "this Y". Those referring expressions are clearer. In this case, "The application", or "The linked repository" would be better.